src/coreclr/vm/codeman.cpp

```diff
@@ -1525,7 +1525,16 @@ void EEJitManager::SetCpuInfo()
 
     if (((cpuFeatures & ARM64IntrinsicConstants_Sve) != 0) && CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_EnableArm64Sve))
     {
-        CPUCompileFlags.Set(InstructionSet_Sve);
+        int maxVectorLength = (maxVectorTBitWidth >> 3);
```

Comment:
nit, the following is much more readable/understandable and modern compilers already optimize it to `>> 3` since `maxVectorTBitWidth` is unsigned:
```suggestion
        uint32_t maxVectorLength = maxVectorTBitWidth / 8;
```

Reply:
Done